packages/pluggableWidgets/color-picker-web/src/components/ColorPicker.tsx

```diff
@@ -95,12 +96,20 @@ export const ColorPicker = (props: ColorPickerProps): ReactElement => {
                 color={colorValue}
                 disabled={disabled}
                 onKeyUp={setColorPickerHidden}
-                onChange={({ target }) => submitColor(target.value)}
+                onChange={({ target }) => {
+                    submitColor(target.value);
+                    props.onChange();
```

Comment:
Wouldn't it make sense to call `onChangeComplete` here instead?

Reply:
We can't, they share `onChange` action but the purpose is different. Also properties are different. So it's better to keep them separated.